src/studio/src/designer/backend.Tests/Controllers/DataModelsController/NonAuthenticatedCallsTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Altinn.Studio.Designer.Configuration;
+using Altinn.Studio.Designer.Controllers;
+using Altinn.Studio.Designer.Services.Interfaces;
+using Designer.Tests.Controllers.ApiTests;
+using Designer.Tests.Mocks;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace Designer.Tests.Controllers.DataModelsController;
+
+public class NonAuthenticatedCallsTests: ApiTestsBase<DatamodelsController, NonAuthenticatedCallsTests>
+{
+    private const string VersionPrefix = "/designer/api";
+    private readonly WebApplicationFactory<DatamodelsController> _factory;
+
+    public NonAuthenticatedCallsTests(WebApplicationFactory<DatamodelsController> factory) : base(factory)
+    {
+        _factory = factory;
+    }
+
+    protected override void ConfigureTestServices(IServiceCollection services)
+    {
+        services.Configure<ServiceRepositorySettings>(c =>
+            c.RepositoryLocation = TestRepositoriesLocation);
+        services.AddSingleton<IGitea, IGiteaMock>();
+    }
+
+    [Fact]
+    public async Task GetDatamodels_NotAuthenticated_ShouldReturn401()
+    {
+        const string url = $"{VersionPrefix}/ttd/hvem-er-hvem/Datamodels/";
+        var httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, url);
+
+        var response = await HttpClient.Value.SendAsync(httpRequestMessage);
+
+        Assert.Equal(HttpStatusCode.Found, response.StatusCode);
```

Comment:
This test is ported from DatamodelsControllerTests class.
It seems wrong. Doesn't seems like an expected behavior. But test was there for a long time and green.
If we perform unauthorized http call response should have HttpStatusCode.Unauthorized status code.
@RonnyB71 @nkylstad  Can you please review and create task related to issue.

Reply:
Hm, I see Found is 302 which previously was Moved Temporarily forcing the client to to a temp redirect. Could there be some handling of this on the frontend? Other than that, I agree, Unautorized seems to be a better response.